problems/problem_simgan.py

```diff
@@ -84,24 +86,32 @@ def construct_dataset(self):
         self.validating_datalist = [simganData.SimGANDataset(real_size=int((128**2)/4), sim_size=128, batch_size=128)]
         # import pdb; pdb.set_trace()
 
-    def objective_functions(self, indiv):
+    def objective_functions(self, population):
```

Comment:
This scores the entire population at once using the tournament evaluation method. This is similar to what we do with the individual networks to find the best refiner/discriminator.